elements/pfe-cta/demo/index.html

```diff
@@ -75,7 +75,7 @@ <h3 slot="pfe-card--header"><code>pfe-color="base"</code></h3>
           </div>
           <div class="event-registry" slot="pfe-card--footer"></div>
         </pfe-card>
-        <pfe-card pfe-color="lighter" pfe-size="small">
+        <pfe-card pfe-color="lightest" pfe-size="small">
```

Comment:
I originally set it to lighter so we can see the CTAs in the gray context and make sure things are readable and didn't break.  It seems like we're taking away a test case from the demo page by using the same color as the other card.

Reply:
@castastrophe  Let me know if this is still relevant, since lighter now is the equivalent of base.